aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -341,7 +341,7 @@ def generate_enhanced_lambda_metrics(log, tags_cache):
     return parsed_metrics
 
 
-def parse_lambda_tags_from_arn(arn):
+def parse_lambda_tags_from_arn(arn, aws_account_tag=False):
```

Comment:
Are you using `aws_account_tag` for something?

Reply:
oops no, I meant to remove that, thanks :)